<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomDeductionsWizard</class>
 <widget class="QWizard" name="CustomDeductionsWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Custom Deductions</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>Custom Deductions</string>
   </property>
   <property name="subTitle">
    <string>Please select the employees to enter dedutions for.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Issue Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dtpIssueDate">
        <property name="displayFormat">
         <string>dd MMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Payroll Month:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboMonth"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Deduction Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboDeductionType">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Funeral</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fundraising</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Other</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Amount Per Employee:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spinAmount">
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="lstEmployees">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="headerHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Roll #</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="title">
    <string>Deduction Details</string>
   </property>
   <property name="subTitle">
    <string>Please enter the deductions issued for each employee:</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableWidget" name="tblDeductions">
      <property name="columnCount">
       <number>5</number>
      </property>
      <column>
       <property name="text">
        <string>Roll No</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount Issued</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>EmployeeID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <property name="title">
    <string>Finish</string>
   </property>
   <property name="subTitle">
    <string>Please click on the save button to save the deductions.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="text">
       <string>Save</string>
      </property>
      <property name="description">
       <string>Click here to save the deductions</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>277</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
